### DISCUSSION TITLE
Tow Away Zone

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#73] [Data Specification]
Should we support designating a Curb Zone as a tow away zone? CDS isn't opinionated on how enforcement of violations occurs, but a municipality that's digitizing curb inventory might be interested in flagging areas as such. I think it makes sense to represent this attribute on the level of a [Rule](https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#rule). Towing is the result of violating a specific Policy's Rules and we already reflect `rate` on the `Rule` object.

**Comment by zanebclark** [comment]
@kenyaw or @jlarsonOmahaNE, how are you planning on representing tow-away zones in your respective cities? Is there a way to reflect this attribute in CDS that I'm missing?

**Reply by jlarsonOmahaNE** [reply]
Would updating the `no stopping` activity description to include no stopping, tow away work? This may also help out with #64

**Reply by kenyaw** [reply]
I agree with Jacob, as our `no stopping` regulatory signage in SF includes a tow away warning. 

> Would updating the `no stopping` activity description to include no stopping, tow away work? This may also help out with #64

**Comment by schnuerle** [comment]
Since CDS does not capture what happens when there is a violation, and what the repercussions are, this is a good start of discussion. Likely this would go in the release after CDS 1.0. 

Some of this could be part of a larger idea of city-wide policies around all ROW travel, and potentially adding this across your CDS feed, much like some of MDS Policy, as opposed to specific curb Zones. Or pricing/rules for geographic zones around a city. Rules for hooded meters.

**Reply by zanebclark** [reply]
@schnuerle , this is a good example of a discussion without a concrete proposal. Unfortunately, I don't have a good feel for how to articulate what you've proposed here.

**Reply by jlarsonOmahaNE** [reply]
Could it be an array of "enforcement signage" in curb Rules then based on that, a rater could be associated to it?
